### ISSUE TITLE
.z file unprotected read permissions

### ISSUE THREAD

**Issue by L0laapk3** [agkozak/zsh-z#92]
Hi, by default the .z file should have read permissions only for the owner, not for group or others. Similar to .zsh_history. I see this as a security issue.

**Comment by agkozak** [comment]
@L0laapk3 I agree entirely, so that is the approach I took in Zsh-z v2.0.0, which I just released. Thanks for your suggestion.
